src/widgets/_checkbuttons.scss

```diff
@@ -62,6 +62,15 @@ checkbutton {
             margin-right: rem(6px);
         }
     }
+
+    &.image-button:checked {
```

Comment:
@danrabbit did you also want to bring the border back? It looks like adding it here works. In the inspector I was playing with this, and thought it looked nice:

```css
radiobutton.image-button {
  border: 1px solid transparent;
}

radiobutton.image-button:checked {
  border: 1px solid alpha(@accent_color, 0.25);
}
```

![Screenshot from 2020-06-29 12-51-21@2x](https://user-images.githubusercontent.com/611168/86044420-4e55ba80-ba07-11ea-94fe-5217b2f0dba8.png)

![Screenshot from 2020-06-29 12 51 42@2x](https://user-images.githubusercontent.com/611168/86044436-5281d800-ba07-11ea-8754-d5f058e97d1d.png)